content/300-guides/200-deployment/001-deploy-prisma.mdx

```diff
@@ -15,20 +15,26 @@ Projects using Prisma Client can be deployed to many different cloud platforms.
 
 Each paradigm has different tradeoffs that affect the performance, scalability, and operational costs of your application.
 
-Moreover, the user traffic pattern of your application is also an important factor to consider. For example, any application with consistent user traffic may be better suited for a [continuously running paradigm](#long-running-process-paas), whereas an application with sudden spikes may be better suited to [serverless](#serverless-faas).
+Moreover, the user traffic pattern of your application is also an important factor to consider. For example, any application with consistent user traffic may be better suited for a [continuously running paradigm](#long-running-process-paas), whereas an application with sudden spikes may be better suited to [serverless](#serverless-functions-faas).
 
 ### Long-running process (PaaS)
 
-Your Node.js process is continuously running and handles multiple requests. Your application can be deployed to a Platform-as-a-Service like [Heroku](./deployment-guides/deploying-to-heroku), [Koyeb](./deployment-guides/deploying-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine.
+Your application is [traditionally deployed](./traditional) if a Node.js process is continuously running and handles multiple requests at the same time. Your application can be deployed to a Platform-as-a-Service like [Heroku](./traditional/deploy-to-heroku), [Koyeb](./traditional/deploy-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine or good old bare metal server.
```

Comment:
> or good old bare metal server.

Is it still PaaS if it's a bare metal server?

Reply:
Good point. Going to update this.

Reply:
Good fix in the commit 👍